pkgs/applications/editors/jetbrains/readme.md

```diff
@@ -19,7 +19,7 @@ To test the build process of every IDE (as well as the process for adding plugin
  - This will update binary IDEs and plugins, and automatically commit them
  - Source builds need a bit more effort, as they **aren't automated at the moment**:
    - Find the build of the stable release you want to target (usually different for pycharm and idea, should have three components)
-   - I find this at https://jetbrains.com/updates/updates.xml (search for `product name="`, then `fullNumber`)
+   - Build number is avaliable at https://www.jetbrains.com/idea/download/other.html for idea and https://www.jetbrains.com/pycharm/download/other.html for pycharm
    - Update the `buildVer` field in source/default.nix
    - Empty the `ideaHash`, `androidHash`, `jpsHash` and `restarterHash` (only `ideaHash` and `restarterHash` changes on a regular basis) fields and try to build to get the new hashes
    - Run `nix build .#jetbrains.(idea/pycharm)-community-src.src.src`, then `./source/build_maven.py source/idea_maven_artefacts.json result/`
```

Comment:
Does this line need updating to reflect the new `pycharm_maven_artefacts.json`?

If so, I wonder if it'd benefit from a nested list format?

```md
- Run
  - `nix build .#jetbrains.idea-community-src.src.src && ./source/build_maven.py source/idea_maven_artefacts.json result/`
  - `nix build .#jetbrains.pycharm-community-src.src.src && ./source/build_maven.py source/pycharm_maven_artefacts.json result/`
```